ebpfapi/rpc_client.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include <ctype.h>
+#include <iostream>
+#include <stdio.h>
+#include <stdlib.h>
+#include <windows.h>
+#include "ebpf_api.h"
+#pragma warning(push)
+#pragma warning(disable : 4100) // 'identifier' : unreferenced formal parameter
+#pragma warning(disable : 4244) // 'conversion' conversion from 'type1' to
+                                // 'type2', possible loss of data
+#undef VOID
+#pragma warning(pop)
+#include "ebpf_windows.h"
+#include "rpc_interface_c.c"
+
+#pragma comment(lib, "Rpcrt4.lib")
+
+static RPC_WSTR _string_binding = nullptr;
+static const WCHAR* _protocol_sequence = L"ncalrpc";
+static bool _binding_initialized = false;
+
+ebpf_result_t
+ebpf_rpc_verify_program(ebpf_program_verify_info* info, const char** logs, uint32_t* logs_size)
+{
+    ebpf_result_t result;
+
+    RpcTryExcept { result = ebpf_client_verify_program(info, logs_size, const_cast<char**>(logs)); }
+    RpcExcept(RpcExceptionFilter(RpcExceptionCode()))
+    {
+        // TODO: (Issue# 247) Add tracing for the RpcExceptionCode() that is returned.
+        result = EBPF_RPC_EXCEPTION;
+    }
+    RpcEndExcept
+
+        return result;
+}
+
+ebpf_result_t
+ebpf_rpc_load_program(ebpf_program_load_info* info, const char** logs, uint32_t* logs_size)
+{
+    ebpf_result_t result;
+
+    RpcTryExcept { result = ebpf_client_verify_and_load_program(info, logs_size, const_cast<char**>(logs)); }
+    RpcExcept(RpcExceptionFilter(RpcExceptionCode()))
+    {
+        // TODO: (Issue# 247) Add tracing for the RpcExceptionCode() that is returned.
+        result = EBPF_RPC_EXCEPTION;
+    }
+    RpcEndExcept
+
+        return result;
+}
+
+RPC_STATUS
+initialize_rpc_binding()
+{
+    RPC_STATUS status;
+
+    status =
```

Comment:
```suggestion
    RPC_STATUS status =
```
can be combined like you did on line 80